test/firtool/instchoice.fir

```diff
@@ -0,0 +1,32 @@
+; RUN: firtool %s --parse-only | FileCheck %s
+
+FIRRTL version 4.0.0
+circuit Foo:
+  option Platform:
+    FPGA
+    ASIC
```

Comment:
Interesting. Yes, the `case` keyword was probably adding nothing here. I'm good with this.